scripts/iso-manager.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
```

Comment:
A little more info in the commit message would be welcome (what the script is doing for instance: by reading the code it seems to auto-mount stuff and doing things after)